### ISSUE TITLE
NPM dependency aws-sdk@2.5.0 causing deployment error

### ISSUE THREAD

**Issue by gmadole** [serverless/serverless#1876]
The latest npm aws-sdk (v2.5.0) is causing errors when deploying.  
If I go back to v2.4.x everything works as expected.

##### Serverless Framework Version: 
1.0.0-beta.1

##### Operating System: 
OSX

##### Expected Behavior:
To deploy to CF

##### Actual Behavior:
Fails to deploy to CF

CF shows
Status: CREATE_FAILED	
Type: AWS::Lambda::Function
Status reason: Error occurred while GetObject. S3 Error Code: InvalidToken. S3 Error Message: The provided token is malformed or otherwise invalid.

##### Stack-Trace (Optional):

```Serverless: Zipping service...
Serverless: Uploading .zip file to S3...
Serverless: Updating Stack...
Serverless: Checking stack update progress...
Serverless: Checking stack update progress...
 
  Serverless Error ---------------------------------------
 
     An error occurred while provisioning your cloudformation:
     The following resource(s) failed to create: [authorizerjwt,
     noop, users, devices]. 
 
  Stack Trace --------------------------------------------
 ```
ServerlessError: An error occurred while provisioning your cloudformation: The following resource(s) failed to create: [authorizerjwt, noop, users, devices]. 
    at /usr/local/opt/nvm/versions/node/v4.3.2/lib/node_modules/serverless/lib/plugins/aws/deploy/lib/updateStack.js:56:33
    at tryCatcher (/usr/local/opt/nvm/versions/node/v4.3.2/lib/node_modules/serverless/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/usr/local/opt/nvm/versions/node/v4.3.2/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:504:31)
    at Promise._settlePromise (/usr/local/opt/nvm/versions/node/v4.3.2/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:561:18)
    at Promise._settlePromise0 (/usr/local/opt/nvm/versions/node/v4.3.2/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:606:10)
    at Promise._settlePromises (/usr/local/opt/nvm/versions/node/v4.3.2/lib/node_modules/serverless/node_modules/bluebird/js/release/promise.js:685:18)
    at Async._drainQueue (/usr/local/opt/nvm/versions/node/v4.3.2/lib/node_modules/serverless/node_modules/bluebird/js/release/async.js:138:16)
    at Async._drainQueues (/usr/local/opt/nvm/versions/node/v4.3.2/lib/node_modules/serverless/node_modules/bluebird/js/release/async.js:148:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/usr/local/opt/nvm/versions/node/v4.3.2/lib/node_modules/serverless/node_modules/bluebird/js/release/async.js:17:14)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
 
  Get Support --------------------------------------------
     Docs:          v1.docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
```

**Comment by flomotlik** [comment]
@gmadole thanks for reporting. Did you define the dependency in a shrink-wrap file? What is the events that Cloudformation is showing as failing?

**Comment by gmadole** [comment]
@flomotlik I just tried reproducing again but this time had no issues...strange.  Maybe this is a non-issue.  I wish I kept the shrinkwrap file from earlier.


For what its worth, this is the CF error:
<img width="1053" alt="screen shot 2016-08-17 at 7 52 05 am" src="https://cloud.githubusercontent.com/assets/518019/17736852/9613e9bc-644f-11e6-8d2a-dab6f10d5fce.png">
When the deployments failed what is interesting is that the lambda .zip was uploaded to S3.
